src/main/java/edu/ucsb/courses/controllers/BasicSearchController.java

```diff
@@ -30,15 +33,29 @@
     UCSBCurriculumService ucsbCurriculumService;
 
     @GetMapping(value = "/basicsearch", produces = "application/json")
-    public ResponseEntity<String> basicsearch(
-        @RequestParam String qtr, 
-        @RequestParam String dept,
-        @RequestParam String level) 
-        throws JsonProcessingException {
+    public ResponseEntity<String> basicsearch(@RequestParam String qtr, @RequestParam String dept,
+            @RequestParam String level) throws JsonProcessingException {
 
         String body = ucsbCurriculumService.getJSON(dept, qtr, level);
-
+        
         return ResponseEntity.ok().body(body);
     }
 
-}
+    // @GetMapping(value = "/basicsearch/downloadcsv", produces = "text/csv")
+    // public void downloadcsv(@RequestParam String qtr, @RequestParam String dept,
+    //         @RequestParam String level, HttpServletResponse response) throws JsonProcessingException {
+
+
+    //     // DownloadCsvService service = new DownloadCsvService();
+    //     String coursePageJson = ucsbCurriculumService.getJSON(dept, qtr, level);
+        
+    //     String filename = String.format("courses-%s-%s-%s.csv", qtr,dept,level);
+    //     response.setContentType("text/csv");
+    //     response.setHeader("Content-Disposition", "attachment; filename=\"" + filename + "\"");
+    //     try {
+    //         downloadCsvService.listToCSV(coursePageJson, response.getWriter());
+    //     } catch (IOException e) {
+    //         logger.error("Error Writing to Response Stream{}", e);
+    //     }
+    // }     
+}
```

Comment:
Is it necessary to keep this commented code in the final

Reply:
^^^ What he said